src/main/java/com/datatorrent/netlet/OptimizedEventLoop.java

```diff
@@ -75,9 +75,9 @@ public boolean add(SelectionKey key)
       }
       keys[pos++] = key;
       if (pos == keys.length) {
-        SelectionKey[] keys = new SelectionKey[this.keys.length << 1];
-        System.arraycopy(this.keys, 0, keys, 0, pos);
-        this.keys = keys;
+        SelectionKey[] lKeys = new SelectionKey[this.keys.length << 1];
```

Comment:
This seems to be the only place in netlet where `l` will be used to avoid shadowing. Will it be better to name it `newKeys` or `reallocatedKeys` and drop `this` as it is not necessary anymore?